### PR TITLE
When customers purchase and authorize, we add customer detail information (name, email, IP) to a transaction

### PR DESCRIPTION
**Pull request by rockyhakjoong**
When customers purchase and authorize, we add customer detail information (name, email, IP) to a transaction.

### CODE REVIEW

**Comment by pi3r** [comment]
@rockyhakjoong Can you update the description of the pull request with a short explanation of the change and also add an entry in the Changelog?

Will review then. Thank you!

**Comment by rockyhakjoong** [comment]
> @rockyhakjoong Can you update the description of the pull request with a short explanation of the change and also add an entry in the Changelog?
> 
> Will review then. Thank you!

I updated the description and title
Thank you.

**Comment by pi3r** [comment]
Thank you, can you update the changelog in the Head section too? https://github.com/activemerchant/active_merchant/blob/master/CHANGELOG#L6.

**Comment by rockyhakjoong** [comment]
> Thank you, can you update the changelog in the Head section too? https://github.com/activemerchant/active_merchant/blob/master/CHANGELOG#L6.

Hello, I just updated for CHANGELOG. 
Thank you.

File: CHANGELOG [hunk 1]
```diff
@@ -100,6 +100,7 @@
 
 == Version 1.114.0
 * BlueSnap: Add address1,address2,phone,shipping_* support #3749
+* Netbanx: Add-customer-information(name,email,IP)-to-a-transaction [rockyhakjoong] #3754
```
Comment (pi3r) [inline CHANGELOG:103]: Can you add that to the `=== HEAD` section

Comment (rockyhakjoong) [reply CHANGELOG]: I updated it. thank you.

**Review by pi3r — APPROVED** [review]

**Pull request merged**